include/deal.II/grid/tria_accessor.templates.h

```diff
@@ -2233,9 +2238,10 @@ TriaAccessor<0, dim, spacedim>::operator ++ () const
 template <int dim, int spacedim>
 inline
 void
-TriaAccessor<0, dim, spacedim>::operator -- () const
+TriaAccessor<0, dim, spacedim>::operator -- ()
 {
-  Assert (false, ExcNotImplemented());
+  if (global_vertex_index!=numbers::invalid_unsigned_int)
+    --global_vertex_index;
```

Comment:
I think you should be explicit here in making sure that if you start with an index of zero, the previous element is `numbers::invalid_unsigned_int`. That's true, by accident, but far from obvious.